lib/hobo_fields/model.rb

```diff
@@ -280,8 +286,7 @@ def attr_type(name)
       def column(name)
         defined?(@table_exists) or @table_exists = table_exists?
         if @table_exists
-          name = name.to_s
-          columns.find { |c| c.name == name }
```

Comment:
This could be really slow! That `find` was a linear search over the columns, matching by name. Rails has had `columns_hash` since 3.x at least.